### PR TITLE
store management

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- ticket redmine: https://edu-redmine.sun-asterisk.vn/issues/76112
- ticket redmine: https://edu-redmine.sun-asterisk.vn/issues/76113

## WHAT (optional)
- Admin view store list.
- User view store list.
- Admin create/update/delete store.

## Evidence (Screenshot or Video)
image
image
image
image
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/commons/dtos/createStore.dto.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  @IsString()
+  @MinLength(MIN_ADDRESS_LENGTH, {
+    message: (args: ValidationArguments) =>
+      t(Error.ADDRESS_MIN_LENGTH, { length: MIN_ADDRESS_LENGTH }),
+  })
+  address: string;
+
+  @IsNumberString()
+  @MinLength(10, {
```
Comment (huongnt-2545) [inline src/commons/dtos/createStore.dto.ts]: define constant trong file e nha

**Review by huongnt-2545 — COMMENTED** [review]
c có comment nhỏ nhé

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**